pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1198,3 +1202,91 @@ func TestFairEvaluationForNodes(t *testing.T) {
 		}
 	}
 }
+
+func TestPreferNominatedNodeFilterCallCounts(t *testing.T) {
+	tests := []struct {
+		name                  string
+		feature               featuregate.Feature
```

Comment:
prefer a boolean and always set the feature gate value.

If a test is meant to have different combinations of feature gates, this makes sense. But this test is specific to this gate.

Reply:
Great suggestion, done!